### PR TITLE
Add verticalAlign property

### PR DESCRIPTION
**Pull request by amberroy**
verticalAlign puts the origin at the bottom, middle (default), or top of
the Altspace enclosure.  Added example than sets verticalAlign='bottom'
and also shows how to load OBJ and COLLADA models.

### CODE REVIEW

File: examples/test-vertical-align/index.html [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Aframe Altspace Component - Test Vertical Align</title>
+    <meta name="description" content="Models — A-Frame">
+    <script src="../build.js"></script>
+  </head>
+  <body>
+    <!-- This scene copied from Aframe examples -->
+    <a-scene altspace='verticalAlign: bottom' >
```
Comment (brianpeiris) [inline examples/test-vertical-align/index.html]: Use double quotes for attributes.

**Comment by brianpeiris** [comment]
You forgot to add the new example to the examples index page.
Looks good otherwise! :+1:

**Comment by brianpeiris** [comment]
Oh and the skybox in the example conflicts with most of the skyboxes in Altspace. Maybe just scale it down a bit.

**Comment by amberroy** [comment]
Added example to the examples index page, also updated README

**Pull request merged**